fast_and_reliable.py

```diff
@@ -0,0 +1,37 @@
+import time
+from typing import IO
+
+
+def fast_and_reliable(file: IO[str]):
+    """
+    This function receive file to read and read all the words in it to list and set, then check the average time take
```

Comment:
This function receives a file to read, and reads all of the words from inside it into a list and into a set. Then, it checks the average time it takes to find a given word in the list/set.